src/System.Private.ServiceModel/src/System/ServiceModel/Channels/AddressingVersion.cs

```diff
@@ -22,6 +22,7 @@ public sealed class AddressingVersion
         private string _defaultFaultAction;
         private const string AddressingNoneToStringFormat = "AddressingNone ({0})";
         private const string Addressing10ToStringFormat = "Addressing10 ({0})";
+        private const string Addressing200408ToStringFormat = "Addressing200408 ({0})";
```

Comment:
This is in a resource file which doesn't look to be on reference source. Your guess is correct, it's also only used in ToString and isn't used for anything sent across the network.